menagerie/concept-shapes/scripts/mint_tagged_union_java.py

```
@@ -0,0 +1,336 @@
+        }
+    )
+
+    # Stability check: mint realization a second time and compare.
+    print("\n[STABILITY] Re-minting realization for byte-stability check...")
+    stable = True
+    check_cid = compute_cid(real_memento)
+    if check_cid != real_cid:
+        print(f"  UNSTABLE: realization: {check_cid} != {real_cid}")
+        stable = False
+    else:
+        print(f"  STABLE: realization: ok")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)